### PR TITLE
Calendly Block: Add alignment options

### LINKED ISSUE #14367: Calendly Block: add alignment support
It would be nice to allow site owners to align the Calendly blocks using the alignment options offered by the block editor.

### PR DESCRIPTION
**Pull request by scruffian**
Fixes #14367

#### Changes proposed in this Pull Request:

* Adds alignment options

#### Testing instructions:
* Add a calendly block and set the style to "link".
* Try each of the alignment options
* Check that they behave as expected in the editor and the front end:
Screenshot 2020-01-21 at 15 30 58
Screenshot 2020-01-21 at 15 30 54

* Do the same for the inline version of the block.

#### Proposed changelog entry for your changes:

* no changelog

### CODE REVIEW

File: extensions/blocks/calendly/attributes.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ const colourValidator = value => hexRegex.test( value );
 const urlValidator = url => ! url || url.startsWith( 'https://calendly.com/' );
 
 export default {
+	align: {
```
Comment (jeherve) [inline extensions/blocks/calendly/attributes.js]: I don't know if we need this, since we don't set a default value.

**Review by jeherve — CHANGES_REQUESTED** [review]
I wonder if we could simplify this and only do this:
https://github.com/Automattic/jetpack/blob/fix/14367/extensions/blocks/calendly/index.js#L31

We would then let all the styling to the themes, since `Jetpack_Gutenberg::block_classes` handles adding the necessary classes to the container around the block when you pick an alignment:
https://github.com/Automattic/jetpack/blob/5b409658faa273297a21f7a2369805bcf8794fe5/extensions/blocks/calendly/calendly.php#L49

File: extensions/blocks/calendly/attributes.js (hunk 1 above)
Comment (scruffian) [reply extensions/blocks/calendly/attributes.js]: We do, without it, the alignment attribute doesn't work.

**Comment by scruffian** [comment]
I'm happy to not add the additonal CSS but I'm not sure if that's very useful. I'd be curious to get feedback from someone on core.

File: extensions/blocks/calendly/edit.js [hunk 2]
```diff
@@ -244,10 +244,10 @@ export default function CalendlyEdit( { attributes, className, clientId, setAttr
 	);
 
 	return (
-		<>
+		<div className={ className }>
```
Comment (scruffian) [inline extensions/blocks/calendly/edit.js]: I thought it would be good to leave this in

**Comment by scruffian** [comment]
I removed the wide alignment options

File: extensions/blocks/calendly/attributes.js (hunk 1 above)
Comment (scruffian) [reply extensions/blocks/calendly/attributes.js]: Actually I checked this again, and you're right we don't need it. I've removed it.

**Review by jeherve — APPROVED** [review]

**Comment by scruffian** [comment]
r201955-wpcom

**Pull request merged**